src/rtapi/rtapi_pci.h

```diff
@@ -104,7 +104,6 @@ int pci_disable_device(struct pci_dev *dev);
           pci_resource_start((dev), (bar)) + 1))
 
 void __iomem *pci_ioremap_bar(struct pci_dev *pdev, int bar);
-inline void iounmap(volatile void __iomem *addr);
```

Comment:
The failure is line 759 of hm2_pci.c
It includes rtapi_pci.h but now does not get iounmap() defined, which it uses.

AFAIR that job has the CFLAGS set slightly differently, so it fails on various warnings that others pass.

As it is used externally by other components, probably best left as it was.